jest.consig.js: moduleNameMapper is misses stores mapping

<!--

Got a question?
===============
The issue list of this repo is **exclusively** for bug reports and feature requests. Please consider the fact that we cannot support you in the creation of your tests. For such issues regarding implementation, please use the following resources:

- Read the docs: http://quasar-framework.org
- For simple/quick questions ask in the Discord chat: https://discord.gg/5TDhbDg
- For complex questions or requiring help, ask on the forum: http://forum.quasar-framework.org/


Reporting a bug?
================
- Are you sure it isn't already reported? Do a search first! It may have already been answered or even fixed in the development branch (`dev`).

- Are you sure you are reporting to the right repo? If you are not reporting an issue which deals directly with Quasar distributable, then there are [multiple Quasar repos](https://github.com/quasarframework) besides this one:
  -- Quasar CLI: https://github.com/quasarframework/quasar-cli
  -- Quasar Starter Kit (project folder template): https://github.com/quasarframework/quasar-template-default
  -- Electron Wrapper: https://github.com/quasarframework/electron-wrapper
  -- Quasar Play: https://github.com/quasarframework/quasar-play
  -- Documentation Website: https://github.com/quasarframework/quasar-framework.org

- Check if the issue is reproducible with the latest stable version of Quasar CLI. If you are using a pre-release, please indicate the specific version you are using.

- It is **required** that you clearly describe the steps necessary to reproduce the issue you are running into. Issues with no clear repro steps will not be triaged. If an issue labeled "need repro" receives no further input from the issue author for more than 5 days, it will be closed.

- If your issue is resolved but still open, don’t hesitate to close it. In case you found a solution by yourself, it could be helpful to explain how you fixed it.

Have a feature suggestion/request?
=======================
Remove the template from below and provide thoughtful commentary *and code samples* on what this feature means for your product. What will it allow you to do that you can't do today? How will it make current work-arounds straightforward? What potential bugs and edge cases does it help to avoid? etc. Please keep it product-centric.
-->

<!-- BUG REPORT TEMPLATE -->
### Software version

OS: Win10
Node: 16.6.1
NPM: 7.20.3
Any other software related to your bug:

### What did you get as the error?
 FAIL  test/jest/__tests__/MyTest.spec.ts
  ● Test suite failed to run

    Cannot find module 'stores/layout' from 
'src/stores/auth.ts'

### What were you expecting?
find module

### What steps did you take, to get the error?
run a test with pinia stores

### How to fix?
Add the line  `  '^stores/(.*)$': '<rootDir>/src/stores/$1', `
to [jest.config.js](https://github.com/quasarframework/quasar-testing/blob/dev/packages/unit-jest/src/templates/typescript/jest.config.js) > moduleNameMapper
as default.

--- a/packages/unit-jest/src/templates/no-typescript/jest.config.js
+++ b/packages/unit-jest/src/templates/no-typescript/jest.config.js
@@ -46,6 +46,7 @@ module.exports = {
     '^components/(.*)$': '<rootDir>/src/components/$1',
     '^layouts/(.*)$': '<rootDir>/src/layouts/$1',
     '^pages/(.*)$': '<rootDir>/src/pages/$1',
+    '^stores/(.*)$': '<rootDir>/src/stores/$1',
     '^assets/(.*)$': '<rootDir>/src/assets/$1',
     '^boot/(.*)$': '<rootDir>/src/boot/$1',
     '.*css$': '@quasar/quasar-app-extension-testing-unit-jest/stub.css',

--- a/packages/unit-jest/src/templates/typescript/jest.config.js
+++ b/packages/unit-jest/src/templates/typescript/jest.config.js
@@ -62,6 +62,7 @@ module.exports = {
     '^components/(.*)$': '<rootDir>/src/components/$1',
     '^layouts/(.*)$': '<rootDir>/src/layouts/$1',
     '^pages/(.*)$': '<rootDir>/src/pages/$1',
+    '^stores/(.*)$': '<rootDir>/src/stores/$1',
     '^assets/(.*)$': '<rootDir>/src/assets/$1',
     '^boot/(.*)$': '<rootDir>/src/boot/$1',
     '.*css$': '@quasar/quasar-app-extension-testing-unit-jest/stub.css',

